cli/core/hack/choco/README.md

```diff
@@ -0,0 +1,67 @@
+# Using Chocolatey to install the Tanzu CLI
+
+This document describes how to build a Chocolatey package for the Tanzu CLI and how to install it.
+
+## Building the Chocolatey package
+
+Executing the `hack/choco/build_package.sh` script will build the Chocolatey package under `cli/core/hack/choco/_output`.
+The `hack/choco/build_package.sh` script is meant to be run on a Linux machine that has `choco` installed.
+This is most easily done using docker.  Note that currently, the docker images for Chocolatey only support an
+`amd64` architecture.  To facilitate building the package, the new `choco-package` Makefile target has been added
+to `cli/core/Makefile`; this Makefile target will first start a docker container and then run the `hack/choco/build_package.sh` script.
+
+```bash
+$ cd tanzu-framework
+$ cd cli/core
+$ make choco-package
+```
+
+### Content of Chocolatey package
+
+Currently, we build a Chocolatey package without including the actual Tanzu CLI binary.  Instead, when the
+package is installed, Chocolatey will download the CLI binary from Github.  This has to do with distribution
+rights but may be changed in the future.
+
+This current design implies that when we build the Chocolatey package, we target a specific release version
+of the Tanzu CLI.  When a new release is available, the Chocolatey scripts will need to be updated to target
+the new release version.  Please see the "Update release version" section below.
+
+## Installing the Tanzu CLI using the built Chocolatey package
+
+Installing the Tanzu CLI using the newly build Chocolatey package can be done on a Windows machine with `choco`
+installed. First, the Chocolatey package must be uploaded to the Windows machine.
+
+For example, if we upload the package to the Windows machine under `$HOME\tanzu-cli-pkg`, we can then simply do:
+
+```bash
+$ choco install tanzu-cli -y --source "$HOME\tanzu-cli-pkg"
+```
+
+## Update release version
+
+To update the release version that will be part of the Chocolatey package:
+
+1. The release assets you wish to install are already available on the projects page [https://github.com/vmware-tanzu/tanzu-framework/releases](https://github.com/vmware-tanzu/tanzu-framework/releases)
+
+1. Store the `sha256` of this new release in the script located at `cli/core/hack/choco/scripts/chocolateyinstall.ps1`.  
+The `sha256` value can be found in the file named `tanzu-framework-executables-checksums.txt` which can be found
+alongside the release you have chosen at
+[https://github.com/vmware-tanzu/tanzu-framework/releases](https://github.com/vmware-tanzu/tanzu-framework/releases).
+For example:
+
+```diff
+ $url64 = "https://github.com/vmware-tanzu/tanzu-framework/releases/download/${releaseVersion}/tanzu-cli-windows-amd64.zip"
+-$checksum64 = '5bc0bd244f89416537a0cacf43ab192ca816a48860eb92328fc5d4093b6be826'
++$checksum64 = '647cd0021f5b8fe84ab482051d2ddfce7a04699bbe743581b1e3082b8ff36305'
+ $checksumType64 = 'sha256'
+```
```

Comment:
question: don't we have to change the `$releaseVersion = 'v0.26.0'` in the script located at `cli/core/hack/choco/scripts/chocolateyinstall.ps1`?

Reply:
We used to, but the latest version automates the version based on the environment variable `VERSION`